src/components/ArmorModal.js

```diff
@@ -0,0 +1,357 @@
+import React, { Component } from "react";
+import { bindActionCreators } from "redux";
+import { connect } from "react-redux";
+import {
+  Card, withStyles, CardHeader, CardContent,
+  Table, TableHead, TableRow, TableBody, TableCell,
+  Tooltip, CardMedia, CardActions, Button, Typography,
+  FormControl, Select, MenuItem, InputLabel, OutlinedInput,
+  IconButton,
+} from "@material-ui/core";
+import { Close } from "@material-ui/icons";
+import { Formik } from "formik";
+import {
+  Radar, RadarChart, PolarGrid, PolarAngleAxis, PolarRadiusAxis,
+} from "recharts";
+import _ from "lodash";
+import { armorToLoadout } from "store/ducks/Loadouts";
+import { SlidingModal, Panel } from "components/layout";
+import { StyledRecharts } from "screens/Styles";
+
+const styles = () => ({
+  imagePanel: {
+    width: "100px",
+    height: "100px",
+    margin: "auto",
+  },
+  betterTooltip: {
+    fontSize: "16px",
+    backgroundColor: "black",
+  },
+  select: {
+    top: "626px",
+    background: "red",
+  },
+  newTitle: {
+    color: "hsl(205,11%,80%)",
+    fontWeight: 600,
+  },
+});
+
+const panelBtn = {
+  cursor: "pointer",
+  padding: "8px",
+};
+
+class ArmorModal extends Component {
+  state = {
+    swapImage: false,
+  }
+
+  render() {
+    const { swapImage } = this.state;
+
+    const {
+      classes, isOpen, onClose, armorData, loadouts, armorToLoadout,
+    } = this.props;
+
+    const { builds } = loadouts;
+
+    const {
+      name, defense, type, resistances, crafting, skills, assets,
```

Comment:
you should default here just to be save `const { name = "", defend = "" ... } = armrorData`